docs/user-guide/src/bmo/live-iso.md

```diff
@@ -0,0 +1,32 @@
+# Live-ISO
+
+The live-iso API in Metal3 allows to boot BaremetalHost with live-iso image instead of using IPA deploy ramdisk.
```

Comment:
```suggestion
The live-iso API in Metal3 allows booting a BareMetalHost with a live ISO image instead of writing an image to the local disk using the IPA deploy ramdisk.
```